.releaserc.yml

```diff
@@ -0,0 +1,23 @@
+---
+branches:
+    - release-1
+
+plugins:
+    - - '@semantic-release/commit-analyzer'
```

Comment:
&lt;rant&gt;
My thought coming into this analysis: "I hate YAML." I'm not sure that's not still true, but what I think is vexing me more is the configuration schema for this particular config. **Note:** This is not your fault: I do not intend to shoot the messenger. 😉

But why in the world did the folks responsible for this schema decide that each plugin item should be a list, with the first item of the list being the name of the plugin, and the second being its configuration options? It took me _forever_ to pull apart this configuration (and I can partly thank my above-mentioned hatred of YAML for that), to come to the conclusion that this was a super long way around simply making a dictionary where the plugin name is the key and the configuration is its value. Like every other like-minded configuration structure under the sun.

I.e. why did they not make this configuration structure:

```yaml
plugins: 
-
  @semantic-release/commit-analyzer: 
    preset: "conventionalcommits"
    releaseRules: 
    - type: "docs"
      scope: "README"
      release: "patch"
-
  @semantic-release/changelog: 
    changelogTitle: "# Changelog\n\nAll notable changes to this project will be documented in this file."
```

YAML gets the continued shade for things like `- -` to denote a list of lists.
&lt;/rant&gt;

Reply:
I actually have one action item to take away from this rant: Can we convert this to a `.releaserc.json` file instead? While the configuration structure is still convoluted, reading it in JSON is going to make my head explode a lot less than YAML is, every time I have to come in and edit it.

Reply:
Cannot agree more. It takes me quite a time to digest this weird data structure.

Reply:
Good suggestion. Let me do it now.